### ISSUE TITLE
17.1.0 was released, _version.py should not be 17.1.0.dev0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#9083]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#9083|
|Type|defect|
|Created|2017-03-16 06:21:53Z|

In https://github.com/twisted/twisted/blob/trunk/src/twisted/_version.py ,
the version is 17.1.0.dev0

This is wrong, since 17.1.0 has already been released.

According to https://www.python.org/dev/peps/pep-0440/#summary-of-permitted-suffixes-and-relative-ordering
_version.py in Twisted should now contain either:

17.1.0.post0 or

17.2.0.dev0 or

17.1.1.dev0 or

....

<details><summary>Searchable metadata</summary>

```
trac-id__9083 9083
type__defect defect
reporter__rodrigc rodrigc
priority__normal normal
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1489645313615660 1489645313615660
changetime__1506274082339254 1506274082339254
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/745

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @rodrigc|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)|@rodrigc removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_6"></a>|@adiroiban set owner to @rodrigc|
|-|-|

+1 for 17.1.1.dev0

Please see the PR comment. I failed to test this change.
Any direction for how to test this is much appreciated

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_7"></a>|@rodrigc set status to `closed`|
|-|-|

We are past 17.1.0

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)|@rodrigc set status to `reopened`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_9"></a>|@rodrigc set owner to @hawkowl<br>@rodrigc set status to `new`|
|-|-|

After doing a Twisted release, @hawkowl is now bumping the version number in trunk to have *dev* in it:
https://github.com/twisted/twisted/commit/cebdcb9aae69ed9190e46359fb2415b3e2a13dce

So right now the released version is **17.9.0**, and the version in trunk is **17.9.1.dev0**.
That works for me.

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)|@rodrigc set status to `closed`|
|-|-|
